storage/innobase/include/dict0mem.h

```diff
@@ -1430,6 +1430,9 @@ struct dict_index_t {
   {
     online_log= reinterpret_cast<row_log_t*>(this);
   }
+
+  /** SCN of the index */
+	trx_id_t trx_scn;
```

Comment:
The comment should explain the exact purpose of the field. As far as I can tell, this field is not being restored from any persistent data structures. Modifications appear to be scheduled via a call to `trx_t::add_scn_index()` in `row_create_index_for_mysql()`, that is, during DDL operations only.

What is the relationship of `dict_index_t::trx_scn` and `dict_index_t::trx_id`? The latter field is not being restored on database startup, because all transactions will be committed or rolled back at server startup and therefore all indexes after crash recovery will be accessible (unless they are marked as corrupted). As far as I could tell (by adding `__attribute((deprecated))` to the declarations), the only place where these fields matter is `row_merge_is_index_usable()`, whose original purpose was to prevent old read views from using indexes that had been created (or tables rebuilt) after the read view creation. Can we do with just `dict_index_t::trx_id`?